### DISCUSSION TITLE
Separate Websocket Logger

### DISCUSSION THREAD

**Discussion by AliRn76** [Kludex/uvicorn#2381] [Q&A]
Can we have a separate logger for Websocket classes, 
Currently, I want to ignore the `websockets` logs but I don't want to change behavior of `uvicorn.error` and it is not possible as I saw, so can we use another logger for the third-party packages? for example `uvicorn.websocket` or anything

https://github.com/encode/uvicorn/blob/c23cd24e6676ee0638d014d7000af1e6e0996bd6/uvicorn/protocols/websockets/websockets_impl.py#L111

**Comment by Kludex** [comment]
It's part of https://github.com/encode/uvicorn/issues/562.
